### ISSUE TITLE
Use Flutter LazyTable for DevTools FlatTables

### ISSUE THREAD

**Issue by kenzieschmoll** [flutter/devtools#3826]
Tracking https://github.com/flutter/flutter/issues/87370. Once this work is ready, we should test it out in DevTools.

**Comment by kenzieschmoll** [comment]
This issue is a stale feature request from 2022 that depended on a Flutter issue (flutter/flutter#87370). That issue is closed, but DevTools does not use LazyTable. This is not high enough priority to rewrite DevTools tables. If we experience performance issues with the DevTools tables then we can consider rewriting them with this widget to see if we can get improvements. If there is more work to do here, please let us know by filing a new issue with up to date information. Thanks!
